advanced/extending-electron-forge/writing-makers.md

```diff
@@ -38,7 +38,7 @@ The options object is documented in [`MakerOptions`](https://js.electronforge.io
 
 ```javascript
 export default class MyMaker extends MakerBase {
-  async make(opts) {
+  async make (opts) {
```

Comment:
Is this space between function and parameter part of the standard? doesn't seem like we commonly use this across the Electron org.

Reply:
Yea, the majority of these changes just came from running with the `--fix` flag, they're straight from `standard` via `@electron/lint-roller`.

[Here's the same space in the docs in `e/e`](https://github.com/electron/electron/blob/46458ab517aebe8fe9573250c62ee814bf2f67ff/docs/api/protocol.md?plain=1#L274).

You're correct that we don't commonly use it in internal code, but the docs on `e/e` have been linted with `standard` for a long time now, and we also lint the example fiddles (see https://github.com/electron/fiddle/pull/839) with `standard` as well. So basically we don't use it for internal code, but do for external code examples. 🤷